terraform/microceph/README.md

```diff
@@ -0,0 +1,64 @@
+# Terraform module for `microceph`
+
+This is a Terraform module facilitating the deployment of the `microceph` charm,
+using the [Terraform Juju provider](https://github.com/juju/terraform-provider-juju/).
+For more information,
+refer to the provider [documentation](https://registry.terraform.io/providers/juju/juju/latest/docs).
+
+> [!IMPORTANT]
+> This module requires a Juju K8s model to be available.
```

Comment:
Why can't this terraform provide the model?
Is it meant to be using the same model as COS?

Reply:
No terraform module (for charms) provide the model. As per the spec.
No, microceph is a machine charm, hence a machine model.